src/Util.hpp

```diff
@@ -117,10 +118,17 @@ bool create_dir(nonstd::string_view dir);
 nonstd::string_view dir_name(nonstd::string_view path);
 
 // Return true if `suffix` is a suffix of `string`.
+// From: https://stackoverflow.com/a/874160/262458
 inline bool
 ends_with(nonstd::string_view string, nonstd::string_view suffix)
 {
-  return string.ends_with(suffix);
```

Comment:
Latest MSVC does not support https://en.cppreference.com/w/cpp/string/basic_string/ends_with in its c++=latest setting?